src/assets/i18n/ru.json5

```
@@ -1549,257 +1549,208 @@
   
   
   // "browse.comcol.by.author": "By Author",
-  // TODO New key - Add a translation
-  "browse.comcol.by.author": "By Author",
+  "browse.comcol.by.author": "По Автору",
   
   // "browse.comcol.by.dateissued": "By Issue Date",
-  // TODO New key - Add a translation
-  "browse.comcol.by.dateissued": "By Issue Date",
+  "browse.comcol.by.dateissued": "По Дате Выпуска",
```

Эээ..в русском языке в заглавиях только первая буква заглавная. Остальные строчные.
```suggestion
  "browse.comcol.by.dateissued": "По дате выпуска",
```